site/homepage/content/release-notes.md

```diff
@@ -11,6 +11,15 @@ title = "Release Notes"
 * The (experimental) CoAP adapter now supports tracking of request processing using
   OpenTracing.
 * A lorawan provider has been added for the loriot network provider.
+* Hono now supports *auto-provisioning* of devices that authenticate with X.509 client certificates. 
+  To enable it, the *auto-provisioning-enabled* property on the trust anchor 
+  must be set to `true` in the tenant configuration. The protocol adapters add the client certificate to the *get* request to the 
+  Credentials API, if this is enabled for the trust anchor used, so that the device registry can create 
+  credentials (and device registration data) "on-the-fly". 
```

Comment:
IMHO we can remove these sentences as the details are covered by the information referred to below ...